### ISSUE TITLE
No Proper 'Confusion Matrix'

### ISSUE THREAD

**Issue by Sarvin92** [isseu/emotion-recognition-neural-networks#22]
I have successfully trained and validated. But, When I run plot_emotion_matrix, I get an improper Confusion Matrix.. It shows Values only for One 'predicted Emotion' column. 
![cnf_matx_angry](https://user-images.githubusercontent.com/32176631/32018479-4cf09a24-b9ca-11e7-8344-e70b9529b31f.png)

**Comment by ignatiousanto** [comment]
yeah I'm going through the same issue. Its not an issue with the confusion matrix code per say.. it looks like the model just seems to prefer predicting one or two classes and converges there. I have tested it out on a few different datasets and all seems to have the same issue.

**Comment by Sarvin92** [comment]
Have you tried to train and validate by changing the Epochs and iteration? This convergence to specific class might be due to overfitting problem.

**Comment by srlaghm** [comment]
I am facing the same problem.  How can it be solved? 
@Sarvin92  do you solved it?
